### DISCUSSION TITLE
bias_variance_decomposition for multi-output regressor

### DISCUSSION THREAD

**Discussion by Baghapour** [rasbt/mlxtend#901] [Q&A] [answered]
Hi, I would appreciate to know how I can use the bias_variance_decompostion function in mlxtend for a multi-output regressor?
(For instance,  using the function with MultiOutputRegressor(GradientBoostingRegressor()) model)
Thanks a lot

**Comment by rasbt** [comment, marked as answer]
Good question. I am not sure this is currently supported. At least, I  haven't tried :P

**Reply by Baghapour** [reply]
Thanks Sebastian, I hope it will be considered sometime later in mlxtend since it would be great to see also the effect of targets interactions by such an error decompaction.
